trfind/finders/summitpost.py

```diff
@@ -1,27 +1,47 @@
 import urllib
 from urllib.parse import urljoin, urlencode
+import requests
```

Comment:
ha, yeah, I guess as long as we have a dep on `requests` we might as well use it.